extensions/EXT/EXT_buffer_storage.txt

```diff
@@ -408,11 +405,6 @@ Errors
     MAP_PERSISTENT_BIT_EXT included in <access>.
 
 
-Dependencies on GL_EXT_direct_state_access
-
-    If GL_EXT_direct_state_access is not supported, remove all references to
-    NamedBufferStorageEXT.
```

Comment:
Please remove actual references to that function, starting from line 79.

Reply:
Yes. This is an ES-only extension so DSA does not exist here. Note that there are 11 references to that function in the extension text.

Reply:
Added another issue to document this.